_posts/2015-09-16-introduction-of-recommender-algorithms-for-e-commerce.md

```diff
@@ -0,0 +1,65 @@
+
+* **Browsers into buyers:** Visitors to a Web site often look over the site without ever purchasing anything. Recommender systems can help customers find products they wish to purchase.
+
+* **Cross-sell:** Recommender systems improve cross-sell by suggesting additional products for the customer to purchase. If the recommendations are good, the average order size should increase. For instance, a site might recommend additional products in the checkout process, based on those products already in the shopping cart.
+
+* **Loyalty:** In a world where a site’s competitors are only a click or two away, gaining customer loyalty is an essential business strategy. [4,5] Recommender systems improve loyalty by creating a value-added relationship between the site and the customer. Sites invest in learning about their users, use recommender systems to operationalize that learning, and present custom interfaces that match customer needs. Customers repay these sites by returning to the ones that best match their needs. [3]
+
+Hence, it is not surprising that Recommender Systems are essential part of e-commerce solutions in this era. At present day, many recommender algorithms exist which can be employed into recommender systems built for e-commerce. In this blog-post we will have high level overview of the classification of such recommender algorithms.
+
+
+##Algorithms
+
+Recommender Algorithms are mainly classified into two categories as described below:
+
+1. ###Content based recommendation systems
+> Content-based recommendation systems recommend an item to a user based upon a description of the item and a profile of the user’s interests. While a user profile may be entered by the user, it is commonly learned from feedback and/or the interaction of the user on items. A variety of learning algorithms have been adapted to learning user profiles. [1]
+>
+> Algorithms: Relevance feedback, genetic algorithms, neural networks, and the Bayesian classifier are among the learning techniques for learning a user profile. The vector space representation and latent semantic indexing can both be used by these learning methods to represent documents.
+>
+> **Pros:** It does not rely on other user's feedback. Provides purely feature based recommendations. It Does not suffer from cold start.
+>
+> **Cons:** Limited content analysis and limited degree of novelty. Lower recommendation value.
+>
+2. ###collaborative-filtering
+> From all different approaches for obtaining recommendations, the most popular one is by far the collaborative-filtering. This approach makes recommendations by discovering correlation between users of the recommender system. It is a unified approach for discovering the list of potentially interesting items (items not been accessed by the active user) and predicting its relevancy to the active user. The idea is to reach the final recommendation only on the basis of similar users and their actions. [6]
```

Comment:
Does it discover one correlation or many? "Correlation" might be written in plural.